test/blueprints/Do.Test.Blueprints.Service.Application/ConfigurationOverrider/ConfigurationOverriderFeature.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Do.Architecture;
+
+namespace Do.Test.ConfigurationOverrider;
+
+public class ConfigurationOverriderFeature : IFeature
+{
+    public void Configure(LayerConfigurator configurator)
+    {
+        configurator.ConfigureAutomapping(mapping =>
+        {
+            mapping.MemberIsId.Clear();
+            mapping.MemberIsId.Add(member => member.PropertyType == typeof(Guid) && member.Name == "EntityId");
```

Comment:
this override is too much for an example, override sql type of only one a column in `Entity`, e.g., make `String` property's length `500` instead of default `255`, and/or add an index to `Guid` property etc.